FireRender.Maya.Src/frWrap.h

```diff
@@ -1083,7 +1084,7 @@ namespace frw
 		void SetVertexColors(const std::vector<int>& vertexIndices, const std::vector<MColor>& vertexColors, rpr_int indexCount)
 		{
 			std::vector<rpr_float> colors;
-			colors.resize(indexCount);
+			colors.resize(indexCount * 4);
```

Comment:
please move constant "4" as const int variable and use this variable in all cases